### PR TITLE
Change inline var PHPDoc syntax

### PR DESCRIPTION
**Pull request by teohhanhui**
Following symfony/symfony-standard#990

### CODE REVIEW

**Comment by Simperfit** [comment]
👍

File: bin/console [hunk 1]
```diff
@@ -11,9 +11,7 @@ use Symfony\Component\Debug\Debug;
 
 set_time_limit(0);
 
-/**
- * @var Composer\Autoload\ClassLoader $loader
- */
+/** @var \Composer\Autoload\ClassLoader $loader */
```
Comment (amenophis) [inline bin/console]: Use a ```use``` at the top of the class, no ?

File: web/app.php [hunk 2]
```diff
@@ -2,9 +2,7 @@
 
 use Symfony\Component\HttpFoundation\Request;
 
-/*
- * @var Composer\Autoload\ClassLoader
- */
+/** @var \Composer\Autoload\ClassLoader $loader */
```
Comment (amenophis) [inline web/app.php]: same below

File: web/app_dev.php [hunk 3]
```diff
@@ -23,9 +23,7 @@
     exit('You are not allowed to access this file. Check '.basename(__FILE__).' for more information.');
 }
 
-/**
- * @var Composer\Autoload\ClassLoader
- */
+/** @var \Composer\Autoload\ClassLoader $loader */
```
Comment (amenophis) [inline web/app_dev.php]: same as below

File: bin/console (hunk 1 above)
Comment (teohhanhui) [inline bin/console]: I'd prefer that we do not deviate from `symfony/symfony-standard` unnecessarily.

**Comment by dunglas** [comment]
Thanks @teohhanhui

**Pull request merged**